tools/acrnlog/README.rst

```diff
@@ -1,63 +1,88 @@
-``acrnlog``
-===========
+.. _acrnlog:
 
-DESCRIPTION
-###########
-``acrnlog`` is a userland tool to capture ACRN hypervisor log, it runs as an
-SOS service at boot. It captures two kinds of logs:
+acrnlog
+#######
 
-- log of current running;
+Description
+***********
 
-- log of last running if crashed and logs remaining.
+``acrnlog`` is a userland tool used to capture a ACRN hypervisor log. It runs as an
+SOS service at boot, capturing two kinds of logs:
 
-The path to save log files is ``/tmp/acrnog/``, so the log files would be lost
-after reset.
+- log of the currently running hypervisor
+- log of last running hypervisor if it crashed and the logs remain.
 
-USAGE
-#####
-The ``acrnlog`` tool is launched as a service at boot, with 4 1MB log files limited.
-To change the log file limitation:
+Log files are saved in ``/tmp/acrnlog/``, so the log files would be lost
+after a system reset.
 
-- temporary change:
-    Stop the ``acrnlog`` service:
+Usage
+*****
 
- ::
+The ``acrnlog`` tool is launched as a service at boot, and limited to
+supporting four 1MB log files by default.  You can change this log file
+limitation temporarily or permanently.
 
- # systemctl disable acrnlog
 
-Restart ``acrnlog`` running in backgroud with size and number of files.
- For example:
+Temporary log file changes
+==========================
 
- ::
+You can temporarily change the log file setting by following these
+steps:
 
- # acrnlog -n 8 -s 4 &
+1. Stop the ``acrnlog`` service:
 
-Use ``get_loglevel``/``set_loglevel`` to query and change the hypervisor loglevel.
+   .. code-block:: none
 
-The ``mem_loglevel`` controls log to be saved using ``acrnlog``, while
-``console_loglevel`` controls log to output to console. For example:
+      # systemctl disable acrnlog
 
- ::
+2. Restart ``acrnlog``, running in the background, and specify the new
+   number of log files and their size (in MB).  For example:
 
-  ACRN:\>get_loglevel
-  console_loglevel: 2, mem_loglevel: 4
-  ACRN:\>set_loglevel 2 5
-  ACRN:\>get_loglevel
-  console_loglevel: 2, mem_loglevel: 5
+   .. code-block:: none
 
-- permanent change:
-   Edit ``/usr/lib/systemd/system/acrnlog.service`` to attached the ``-n`` and ``-s`` options to the ``ExecStart`` cmd, and restart the service. For example:
+      # acrnlog -n 8 -s 4 &
 
- ::
+You can use ``get_loglevel`` and ``set_loglevel`` to query and change
+the hypervisor loglevel.
 
-  ExecStart=/usr/bin/acrnlog -n 8 -s 4
+The ``mem_loglevel`` command controls the log to be saved using
+``acrnlog``, while the ``console_loglevel`` command controls the log
+output to the console. For example:
 
+.. code-block:: none
 
-BUILD & INSTALL
-##################
+   ACRN:\>get_loglevel
```

Comment:
Where is that running? Is that the ACRN hypervisor shell (and if so, how do you get there)?

Reply:
no idea.  That was what was in the reviewed doc before.

Reply:
Fair enough :-)

@lyan3 , can you answer this question? (it would be good to adjust the doc with the response)

Reply:
I looked it up in the code and confirmed these are commands from the hypervisor shell - created PR #333 to add a note clarifying that.

@dbkinder @lyan3 please take a look

Reply:
Yes, that's right. Thanks for adding the note.